packages/testsweets/pubspec.yaml

```diff
@@ -21,11 +21,7 @@ dependencies:
   logger: ^1.1.0
   recase: ^4.0.0
   flutter_screenutil: ^5.0.0+2
-  
```

Comment:
@FilledStacks  This is the fix that I made to github actions to make it work again, is there a reason to remove it